tests/drivers/i2s/i2s_api/src/test_i2s_loopback.c

```diff
@@ -319,7 +319,7 @@ void test_i2s_transfer_restart(void)
 	TC_PRINT("Stop transmission\n");
 
 	/* Keep interface inactive */
-	k_sleep(TEST_I2S_TRANSFER_RESTART_PAUSE_LENGTH_US);
+	k_sleep(K_MSEC(TEST_I2S_TRANSFER_RESTART_PAUSE_LENGTH_US));
```

Comment:
@mnkp I am unsure whether the initial intention was to to specify these pauses in microseconds or milliseconds?

Reply:
This really doesn't feel right, I will not merge this until @mnkp has had a chance to take a look.

Reply:
The fix proposed by @henrikbrixandersen is correct. However, let's also update the naming of the timeout defines by changing _US suffix to _MS. They were wrong. Or we could drop defines and put the value directly as the parameter as in `k_sleep(K_MSEC(1000))`. Hardcoded timeout values are used in a few more places in the code. The testcase needs refactoring anyway to support more i2s drivers.

Reply:
@mnkp, @carlescufi Thanks. I have updated the compilation fix to remove the defines.